roles/variables/defaults/main.yml

```diff
@@ -700,10 +734,20 @@ kafka_broker_custom_client_properties: {}
 ### Boolean to enable the embedded rest proxy within Kafka. NOTE- Embedded Rest Proxy must be enabled if RBAC is enabled and Confluent Server must be enabled
 kafka_broker_rest_proxy_enabled: "{{confluent_server_enabled and not ccloud_kafka_enabled }}"
 
+### Property of ERP as MDS client. Can be set to true when ssl_client_authentication is not none. When set to true will not send oauth token or ldap creds to MDS even when MDS server has support for accepting them. Keeping false means if MDS has oauth and mtls support client will send both oauth token and cert
+kafka_broker_rest_proxy_to_mds_send_certs_only: false
```

Comment:
this variable naming should be better, it's too long for a variable name. Please rename to something like `rp_mds_certs_only` and add the details in the docs

Reply:
I thought all variables for ERP have the prefix kafka_broker_rest_proxy_ so I also kept the same prefix. Length wise yes its long but there are many longer variables.